src/main/java/response/HttpResponseWriter.java

```diff
@@ -0,0 +1,20 @@
+package com.td.HttpServer;
+import java.io.IOException;
+
+public class HttpResponseWriter {
```

Comment:
As is I can't justify it. It would probably be okay to get ride of the `HttpResponseFormatter` class and put that logic into this class, though. So that this class would format the response and then send it to the `SocketIO`